samples/client/petstore/python-experimental/test/test_child.py

```diff
@@ -64,19 +64,24 @@ def testChild(self):
 
         # setting a value that doesn't exist raises an exception
         # with a key
-        with self.assertRaises(petstore_api.ApiKeyError):
+        with self.assertRaises(AttributeError):
```

Comment:
nit:
We should be checking against our specific petstore_api.ApiAttributeErrors here
not the general python AttributeError

Reply:
I thought about with but actually I think it's better to check for AttributeError because that's exactly what hasattr/getattr/setattr expect. A slight improvement would be to assert for bot exceptions.

Reply:
If we asserted only the AttributeError then we are not proving that we are raising an api.Api AttributeError. It would be more specific to check both or api.Api AttributeError because that is the specific exception that we expect our package to raise.